great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -357,12 +364,12 @@ def dataframe(self):
         ValueError Exception
         """
         # Changed to is None because was breaking prior
-        if self.active_batch_data is None:
+        if self.batch_manager.active_batch_data is None:
             raise ValueError(
                 "Batch has not been loaded - please run load_batch_data() to load a batch."
             )
 
-        return self.active_batch_data.dataframe
+        return cast(PandasBatchData, self.batch_manager.active_batch_data).dataframe
```

Comment:
Can we put a return type of the property here and in other places? I haven't checked but I'm wondering it could also us to remove casts elsewhere.

Reply:
@billdirks Will do (in next commit).  Thanks.